pkg/assembler/assembler_test.go

```diff
@@ -8,6 +8,97 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestAssertEqRegisterToInstrList(t *testing.T) {
+	instrList := parseSingleInstructionToInstrList("[ap] = [fp], ap++;")
+
+	expected := Instruction{
+		OffDest:     0,
+		OffOp0:      -1,
+		OffOp1:      0,
+		DstRegister: 0,
+		Op0Register: 1,
+		Op1Source:   2,
+		Res:         0,
+		PcUpdate:    0,
+		ApUpdate:    2,
+		Opcode:      4,
+	}
```

Comment:
Use the constant values instead of integers please. Apply this for all test cases.  Using constant values instead of numbers makes changes in the code easier, also it is easier for people to read. Like `Op0Register: 1` doesn't same the same as `Op0Register: Fp` for example